src/codecs/imagequant/processor.ts

```diff
@@ -0,0 +1,43 @@
+import imagequant, { QuantizerModule } from '../../../codecs/imagequant/imagequant';
+import wasmUrl from '../../../codecs/imagequant/imagequant.wasm';
+import { QuantizeOptions } from './processor-meta';
+
+let emscriptenModule: Promise<QuantizerModule>;
+
+function initModule(): Promise<QuantizerModule> {
+  return new Promise((resolve) => {
+    const m = imagequant({
+      // Just to be safe, don’t automatically invoke any wasm functions
+      noInitialRun: false,
```

Comment:
Just noticed, this seems to do the opposite of what the comment above indicates.

`no initial run = false` --> `initial run = true`

Reply:
@surma ^